.github/workflows/release.yaml

```
@@ -1,123 +1,241 @@
 
-    - name: Install Cosign
-      uses: sigstore/cosign-installer@main
+    - name: Create SBOM directory
+      id: output
+      shell: bash
+      run: |
+        mkdir -p ${{ github.workspace }}/sbom
 
-    - name: Run Melange
-      uses: chainguard-dev/actions/melange-build@main
+    - if: hashFiles("${{ inputs.config-dir }}/melange.yaml") != ''
```

actionlint (1.7.7)

if: condition "hashFiles(\"${{ inputs.config-dir }}/melange.yaml\") != ''" is always evaluated to true because extra characters are around ${{ }}

(if-cond)